topics/epigenetics/slides/index.html

```diff
@@ -52,7 +52,7 @@
 
 ---
 
-# [Hands-on](https://galaxyproject.github.io/training-material/topics/epigenetics/tutorials/methylation-seq/tutorial.html)
+# [Hands-on]({{site.url}}topics/epigenetics/tutorials/methylation-seq/tutorial.html)
```

Comment:
The reason is `site.url` could be empty (and that would still be valid) but then all your URLs will be relative to the current page, i.e. most of these URLs will be wrong. With a leading slash, you avoid this error as `/...` will always point to the right URL.